staging/velero/Chart.yaml

```diff
@@ -1,19 +1,21 @@
 apiVersion: v1
-appVersion: 1.0.1
+appVersion: 1.4.0
 description: A Helm chart for velero
-home: https://github.com/heptio/velero
-icon: https://cdn-images-1.medium.com/max/1600/1*-9mb3AKnKdcL_QD3CMnthQ.png
-maintainers:
-- email: d-caruso@hotmail.it
-  name: domcar
-- email: hfernandez@mesosphere.com
```

Comment:
probably should keep hector as maintainer, and add jie also, not to mention yourself?